<!--local position rectified by boundary detect-->
<launch>
   <node pkg = "iarc_tf" type = "ned_world_broadcaster" name ="ned_world_brocaster_node"/>
   <node pkg = "iarc_tf" type = "ned_world_transform" name = "ned_world_transform_node"/>
</launch>
